code/datums/brain_damage/psychological.dm

```diff
@@ -0,0 +1,297 @@
+/*TODO:
+	PTSD
+*/
+/datum/brain_trauma/psychological
```

Comment:
Using brain traumas for this won't work if you can just remove them like you can with any other brain trauma.

Reply:
@Qustinnus These traumas get given TRAUMA_RESILIENCE_ABSOLUTE when someone gets them from having a high psych instability value, so you can't use neurine or the like to cure them unless the patient got them from just normal brain damage or something.

Reply:
sounds like a bad work-around of the system because you're introducing a secondary cure system to traumas.

Reply:
Don't just resolve conversations if you havnt addressed the actual issue. You should be reworking brain traumas to work as generic traumas and then make brain traumas a subtype of that.